webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/domain/nodes/Node.scala

```diff
@@ -224,30 +230,30 @@ object CompareProperties {
     }
 
     optNewProps match {
-      case None => Full(oldProps)
+      case None => Right(oldProps)
       case Some(newProps) =>
         val oldPropsMap = oldProps.map(p => (p.name, p)).toMap
 
         //update only according to rights - we get a seq of option[either[remove, update]]
         for {
-          updated <- sequence(newProps) { newProp =>
+          updated <- newProps.toList.traverse { newProp =>
                        oldPropsMap.get(newProp.name) match {
                          case None =>
-                           Full(updateOrRemoveProp(newProp))
+                           Right(updateOrRemoveProp(newProp))
                          case Some(oldProp@NodeProperty(name, value, provider)) =>
                              if(canBeUpdated(old = provider, newer = newProp.provider)) {
-                               Full(updateOrRemoveProp(newProp))
+                               Right(updateOrRemoveProp(newProp))
                              } else {
                                val old = provider.getOrElse(NodeProperty.rudderNodePropertyProvider).value
                                val current = newProp.provider.getOrElse(NodeProperty.rudderNodePropertyProvider).value
-                               Failure(s"You can not update property '${name}' which is owned by provider '${old}' thanks to provider '${current}'")
+                               Left(Inconsistency(s"You can not update property '${name}' which is owned by provider '${old}' thanks to provider '${current}'"))
```

Comment:
Inconsistency here is an inventory error 

Don't you want to use the Inconstancy from com.normation.erros ? 

Maybe We should fi the typo in the class in master/6.1 ...